subsys/net/lib/mqtt/mqtt.c

```diff
@@ -647,3 +647,24 @@ int mqtt_read_publish_payload_blocking(struct mqtt_client *client, void *buffer,
 {
 	return read_publish_payload(client, buffer, length, true);
 }
+
+int mqtt_readall_publish_payload(struct mqtt_client *client, u8_t *buffer,
+				 size_t length)
+{
+	u8_t *end = buffer + length;
+
+	while (buffer < end) {
+		int ret = mqtt_read_publish_payload_blocking(client, buffer,
+							     end - buffer);
+
+		if (ret < 0) {
+			return ret;
+		} else if (ret == 0) {
+			return -EIO;
+		}
+
+		buffer += ret;
+	}
+	return 0;
```

Comment:
Please add empty line after closing }

Reply:
Curious, is there any reason why this is not enforced by CI/checkpatch?

Reply:
Don't know why sanity checker does not check these.